.gitignore

```diff
@@ -1,7 +1,6 @@
 pkg/
 Gemfile.lock
 Gemfile.local
-vendor/
```

Comment:
As per recommendation, users are encouraged to add their DSC modules `/build/vendor/custom`

https://github.com/puppetlabs/puppetlabs-dsc/blob/master/README_BUILD.md#getting-started